meta/acronyms.txt

```diff
@@ -94,6 +96,7 @@ UDP - User Datagram Protocol
 VNI - Virtual Network Interface
 VNID - Virtual Network Identifier
 VR - Virtual Router
+VRF - Virtual Route Forwarding
```

Comment:
I guess it's "Virtual Routing and Forwarding"

Reply:
Merge Diff. Not related to NAT changes

Reply:
@JaiOCP , better to do rebase instead of merge so that other commits won't show up.